python/ray/workers/default_worker.py

```diff
@@ -163,6 +170,7 @@
     if mode == ray.RESTORE_WORKER_MODE or mode == ray.SPILL_WORKER_MODE:
         from ray import external_storage
 
+        ray.storage.impl._init_storage(args.storage, is_head=False)
```

Comment:
What's the difference between `ray.storage.impl._init_storage` and `ray.internal.storage._init_storage`? Why use `ray.storage.impl._init_storage`  here?

Reply:
Fixed.